src/transaction.ts

```diff
@@ -123,6 +126,7 @@ export interface BatchUpdateCallback {
     response?: spannerClient.spanner.v1.ExecuteBatchDmlResponse
   ): void;
 }
+export type BatchUpdateOptions = CommitOptions;
```

Comment:
Does `BatchUpdateOptions` suppose to accept `returnCommitStats`?

Reply:
Also to have in mind that "requestOptions" (or "transactionRequetOptions" or better name) of `BatchUpdateOptions` will support `requestTag` in the future.

Reply:
Once proto changes for tagging is merged something like this could be used
```ts
requestOptions?: Omit<IRequestOptions, 'transactionTag'>
```